### ISSUE TITLE
About the hifiasm version choice

### ISSUE THREAD

**Issue by jiangxiuhan0310** [chhylp123/hifiasm#739]
Hi, thanks for such excellent software for the genome assembly. 
Now, I want to run a genome assembly of diploid animal by using hifiasm, and I just want to use Hifi data and HIC data. I have heard that the new version of Hifiasm has changed the parameters to accommodate HiFi+ONT+HIC assembly. Therefore, I would like to change my Hifiasm to 0.16.1 in order to run the genome assembly.
But I am not pretty sure for that  is right, Could you give me some ideas? Thank you

**Comment by chhylp123** [comment]
Not really. It would be better to always run the latest version of hifiasm.

**Comment by jiangxiuhan0310** [comment]
Thanks for you reply. Much appreciate!
